packages/wrangler/e2e/start-worker-mixed-mode.test.ts

```diff
@@ -0,0 +1,147 @@
+import { readFile, writeFile } from "node:fs/promises";
+import { resolve } from "node:path";
+import { setTimeout } from "node:timers/promises";
+import dedent from "ts-dedent";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { WranglerE2ETestHelper } from "./helpers/e2e-wrangler-test";
+import { generateResourceName } from "./helpers/generate-resource-name";
+
+describe("startWorker - mixed mode", () => {
+	const remoteWorkerName = generateResourceName();
+	const helper = new WranglerE2ETestHelper();
+
+	beforeAll(async () => {
+		await helper.seed(resolve(__dirname, "./seed-files/mixed-mode-workers"));
+		await helper.seed({
+			"remote-worker.js": dedent/* javascript */ `
+					export default {
+						fetch() {
+							return new Response('Hello from a remote worker (startWorker mixed-mode)');
+						}
+					};
+			`,
+		});
+		await helper.run(
+			`wrangler deploy remote-worker.js --name ${remoteWorkerName} --compatibility-date 2025-01-01`
+		);
+	}, 35_000);
+
+	afterAll(async () => {
+		await helper.run(`wrangler delete --name ${remoteWorkerName}`);
+	});
+
+	[true, false].forEach((experimentalMixedMode) => {
```

Comment:
Ideally this should be a `describe.each()` block

Reply:
![Screenshot 2025-06-05 at 12 29 08](https://github.com/user-attachments/assets/1282dcf3-6d71-400c-91d7-0e8d17cfbf85)

Reply:
I'd prefer using `describe.each()` and avoiding inline snapshots, if that's alright? The reason inline snapshots can't be used in a `describe.each()` is because the snapshot can't be updated independently for each case, which is also true here

Reply:
sure I'll make the change 🙂👍

Reply:
updated 🙂👍